odh/base/monitoring/github-receiver-service.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: github-receiver-service
+spec:
+  ports:
+    - name: receiver
```

Comment:
don't you need to add a port number here?

Reply:
As per @tumido suggestion [here](https://github.com/operate-first/apps/pull/68#discussion_r539196386), I'm referring to the port `name` directly (the name specified in `containerPorts` in the deployment spec)

Reply:
He was probably referring to the `targetPort`, the service port itself requires a `port` field, which is the outgoing port. We can wait for @tumido to clarify.

Reply:
Yes, that's what I meant - the `targetPort` Sorry for the confusion, I've inlined the comment on the `targetPort` line and it didn't cross my mind that it can be interpreted as for the whole port object. :smile: 

The `port` property is needed in there, `protocol` is TCP by default and `targetPort` should be either nuber but rather the exported pod's `containerPort` port name.

Reference:
https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/user-guide/services.md#defining-a-service

Reply:
@tumido @HumairAK fixed :+1: